### PR TITLE
Redirects schema

### PR DESCRIPTION
**Pull request by NathHorrigan**

### CODE REVIEW

File: tbx/graphql/schema.py [hunk 1]
```diff
@@ -487,6 +488,13 @@ class Meta:
         interfaces = [PageInterface]
 
 
+class RedirectObjectType(graphene.ObjectType):
+    site = graphene.String()
```
Comment (kaedroho) [inline tbx/graphql/schema.py]: Do we need the ``site`` field?

File: tbx/graphql/schema.py [hunk 2]
```diff
@@ -487,6 +488,13 @@ class Meta:
...
+    old_path = graphene.String()
+    link = graphene.String()
```
Comment (kaedroho) [inline tbx/graphql/schema.py]: What about redirects that link to a page?

File: tbx/graphql/schema.py (hunk 1 above)
Comment (tm-kn) [reply tbx/graphql/schema.py]: no

File: tbx/graphql/schema.py (hunk 2 above)
Comment (tm-kn) [reply tbx/graphql/schema.py]: yes!

File: tbx/graphql/schema.py [hunk 3]
```diff
@@ -699,6 +724,9 @@ def resolve_contact_reasons(self, info):
         except ContactReasonsList.DoesNotExist:
             return None
 
+    def resolve_redirects(self, info):
+        return Redirect.objects.select_related('redirect_page')
```
Comment (zerolab) [inline tbx/graphql/schema.py]: This is missing an `.all()`

File: tbx/settings/dev.py [hunk 4]
```diff
@@ -18,6 +18,7 @@
 
 AUTH_PASSWORD_VALIDATORS = []
 
+PREVIEW_URL = 'http://localhost:8003/preview/'
```
Comment (zerolab) [inline tbx/settings/dev.py]: Can you add a comment with a link to https://github.com/torchbox/torchbox-frontend for context?

File: tbx/graphql/schema.py (hunk 3 above)
Comment (tm-kn) [reply tbx/graphql/schema.py]: Why would you need `all`? I don't understand. :question: :confused:

File: tbx/settings/dev.py (hunk 4 above)
Comment (tm-kn) [reply tbx/settings/dev.py]: This is from Nathan's code. I do not actually understand what it does.

File: tbx/graphql/schema.py (hunk 3 above)
Comment (zerolab) [reply tbx/graphql/schema.py]: `Redirect.objects` is just the manager, whereas all the other resolvers in the schema have either `.all()` or `.live()` which return the queryset. As far as I understand it you want to return the latter, but maybe I am just misreading the django graphene docs

File: tbx/settings/dev.py (hunk 4 above)
Comment (zerolab) [reply tbx/settings/dev.py]: This configure the headless preview URL to point to the FE instance.

File: tbx/graphql/schema.py (hunk 3 above)
Comment (zerolab) [reply tbx/graphql/schema.py]: PEBCAK!

**Pull request merged**